charts/rancher-backup/files/default-resourceset-contents/fleet.yaml

```diff
@@ -1,6 +1,14 @@
+- apiVersion: "management.cattle.io/v3"
```

Comment:
This is already covered by:
https://github.com/rancher/backup-restore-operator/blob/f6209d69089f78de7653301ae3bb0b3fb54bd46d/charts/rancher-backup/files/default-resourceset-contents/fleet.yaml#L34

Reply:
Thanks, I didn't check for the group.